cascade-api/test/Test/Cascade/Api/StateMachine/Command/Task.hs

```diff
@@ -462,7 +465,10 @@ deleteNotExistingById =
         ids -> Gen.element ids |> fmap DeleteById |> Just
 
       execute :: DeleteById Concrete -> m Cascade.Api.Tasks.DeleteByIdResponse
-      execute input = evalIO . Cascade.Api.Tasks.deleteById $ id where id = input ^. #id . concreted
+      execute input = do
+        label "[Task/Delete Non Existing By Id]"
+        let id = input ^. #id . concreted
+        evalIO . Cascade.Api.Tasks.deleteById $ id
```

Comment:
There is no need for this assignment.

We can just pattern match on input and use `concrete` function on id.

``` haskell
execute DeleteById { id } =
  evalIO . Cascade.Api.Tasks.deleteById $ concrete id
```